website/pages/contribute.txt

```diff
@@ -20,7 +20,7 @@ See [[translations]] on how to update translation files.
 The development planning is more or less managed using the [[https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues|bug tracker]]
 
 ===== Patches =====
-Please see our [[https://github.com/zim-desktop-wiki/zim-desktop-wiki|GitHub repository]] for the latest developement code. To submit patches please create your own branch and once done file a merge request.
+Please see our [[https://github.com/jaap-karssenberg/zim-desktop-wiki|GitHub repository]] for the latest development code. To submit patches please create your own branch and once done file a merge request.
```

Comment:
Actually the repository moved, changed the URL on purpose

Reply:
done